packages/api-electron/preload.js

```diff
@@ -23,6 +23,10 @@ class Window {
       features
     });
   }
+
+  static getCurrentWindowId() {
+    return ipc.sendSync('get-window-id');
```

Comment:
Yeah. With the sandbox enabled, there's no way to get the window id from the render process :(